sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -547,8 +554,8 @@ export function fromRheaMessage(
         : undefined,
     ...sbmsg,
     ...props,
-    deadLetterReason: sbmsg.applicationProperties?.DeadLetterReason,
-    deadLetterErrorDescription: sbmsg.applicationProperties?.DeadLetterErrorDescription
+    deadLetterReason: sbmsg.applicationProperties?.DeadLetterReason as string,
+    deadLetterErrorDescription: sbmsg.applicationProperties?.DeadLetterErrorDescription as string
```

Comment:
Why do you need to add `as string`? Couldn't these be undefined?

Reply:
Good point, undefined is allowed.